save-backend/src/main/kotlin/org/cqfn/save/backend/service/LnkUserProjectService.kt

```diff
@@ -44,4 +50,49 @@ class LnkUserProjectService(private val lnkUserProjectRepository: LnkUserProject
             ?: LnkUserProject(project, user, role)
         lnkUserProjectRepository.save(lnkUserProject)
     }
+
+    /**
+     * @param user that should be deleted from [project]
+     * @param project
+     * @return none
+     */
+    @Suppress("UnsafeCallOnNullableType")
+    fun removeRole(user: User, project: Project) = lnkUserProjectRepository.findByUserIdAndProject(user.id!!, project)
+        ?.id
+        ?.let {
+            lnkUserProjectRepository.deleteById(it)
+        }
+        ?: run {
+            logger.warn("Cannot delete user ${user.name ?: user.id} from project ${project.organization.name}/${project.name}: no such link was found.")
```

Comment:
Also throw exception or something similar? Are you currently relying on nullability of return type? I think for `Unit?` it's a bit confusing

Reply:
Ok, now will throw an exception